### PR TITLE
docs(en-US): update intro

### PR DESCRIPTION
**Pull request by gao-sun**
align with the main repo

### CODE REVIEW

File: docs/docs/README.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ slug: /
 
 Main features:
 
-- All-platform user sign-in / sign-up experience with SDK support (SPA, iOS and Android)
-- Passwordless (SMS / Email) support
-- Out-of-box social sign in support (GitHub, Google, WeChat, Alipay, etc.)
+- An OIDC-based identity service
+- All-platform user sign-in/up experience with dark mode and SDKs (Web, iOS, and Android)
+- Sign-in/up with dynamic SMS/Email passcode
```
Comment (IceHe) [inline docs/docs/README.md:14]: better align our term `register`: `Sign-in/up` → `Sign-in/register`?

File: docs/docs/README.md [hunk 2]
```diff
@@ -9,12 +9,12 @@ slug: /
...
+- Out-of-box social sign-in integration (GitHub, Google, WeChat, Alipay, etc.)
```
Comment (IceHe) [inline docs/docs/README.md:15]: nit: `Alipay, etc.` → `Alipay, and etc.`

**Review by IceHe — APPROVED** [review]
Overall LGTM

File: docs/docs/README.md (hunk 1 above)
Comment (gao-sun) [reply docs/docs/README.md:14]: actually we do not use register for content or display. either create account or sign-up

File: docs/docs/README.md (hunk 2 above)
Comment (gao-sun) [reply docs/docs/README.md:15]: image

**Pull request merged**